NewSpring/ApollosProject
# Adds Bitrise Setup

## DESCRIPTION

### What does this PR do, or why is it needed?

Adds some configuration so that Bitrise will build the project correctly. 

### How do I test this PR?

Checkout the Bitrise project and confirm the app is building

![image](https://user-images.githubusercontent.com/1637694/56908252-8f50e880-6a6b-11e9-9a22-6caed7013e7e.png)

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
